workers/dockerfiles/Dockerfile.salmon

```diff
@@ -42,9 +42,10 @@ WORKDIR /home/user
 # Install Salmon
 ENV SALMON_VERSION 0.13.1
 RUN wget https://github.com/COMBINE-lab/salmon/releases/download/v${SALMON_VERSION}/Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz
-RUN tar -xzf Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz
+RUN mkdir Salmon-${SALMON_VERSION}_linux_x86_64
+RUN tar -xzf Salmon-${SALMON_VERSION}_linux_x86_64.tar.gz -C Salmon-${SALMON_VERSION}_linux_x86_64 --strip-components 1
```

Comment:
Can you drop a comment about why this `--strip-components` jawn is necessary?

Reply:
sure, just did!